capstone/src/components/Visitor/Login/Login.jsx

```diff
@@ -32,6 +32,9 @@ export default function Login({ setIsLoggedIn }) {
     setLogin({ ...loginForm, [inputName]: value });
   }
   async function handleSubmitLogIn() {
+    console.log('front');
+    console.log(loginForm.username);
+    console.log(loginForm.password);
```

Comment:
It's OK for now (and I probably won't bring this up again), but in general it's discouraged to commit `console.log`s. One way to avoid this is to be more selective with what you commit by using `git add -p`.

Reply:
Thanks a lot! Won't happen again